src/wp-includes/global-styles-and-settings.php

```diff
@@ -355,7 +355,7 @@ function wp_add_global_styles_for_blocks() {
 			$result = array_values(
```

Comment:
This is a file controlled by gutenberg. This change will need to made in the gutenberg repo and ported across.

Reply:
Note to self, this is the Gutenberg file: https://github.com/WordPress/gutenberg/blob/trunk/lib/global-styles-and-settings.php